mentii/mentii
# Remove self

Allows students to drop a class they are in.

**Significant Changes**
Adds a button to the details page for dropping a class.


**How to Test**
1. Go to a class detail page, if you aren't enrolled you'll see a join button, if you are you'll see a leave button.
2. Click the leave button.
3. Profit.

**Notes**
No unit test is added since I'm using existing functionality, but I did update the existing ones since they weren't calling backend functions correctly. I also touched some random listComponent file to fix someones ladda mistake.

**Quotes**
"Time is money, money is power, power is pizza, and pizza is knowledge."
